Projects/useful_links.md

```diff
@@ -0,0 +1,2 @@
+### Final Year Engineering Projects
+- [Project : Code Begetter](http://blog.thecollegestore.in/project-code-begetter/) - Interesting product “Code Begetter”, that provides a one stop solution for getting an automatic code along with dynamic database just by filling up a few details
```

Comment:
It's seems like it's a just project portfolio or personal experience, which could not be considered as working demo of live project. Project should contain demo or prototype by which viewer can learn about it. Even-though you may add it in [project section](https://github.com/GDGAhmedabad/Learning-Resources/tree/master/Projects) as definition.